lib/project_and_version_value.rb

```diff
@@ -49,8 +49,8 @@ def project_costs(proj)
   # @return [Issue] issue object
   def version_issues(proj_id, version_id)
     proj = Project.find(proj_id)
-    Issue.cross_project_scope(proj, 'descendants')
-      .where('start_date IS NOT NULL AND due_date IS NOT NULL AND fixed_version_id = ? ', version_id)
+    Issue.cross_project_scope(proj, "descendants")
+      .where("start_date IS NOT NULL AND due_date IS NOT NULL AND fixed_version_id = ? ", version_id)
```

Comment:
Line is too long. [101/80]<br>Place the . on the previous line, together with the method call receiver.